extras/userdoc/md/documentation/part-3-connecting-networks-with-synapses.md

```diff
@@ -44,7 +44,7 @@ For the majority of synapses, all of their parameters are accessible via
 `GetDefaults()` and `SetDefaults()`. Synapse models implementing spike-timing
```

Comment:
As before, copy section from part 1 about other sections and the examples (sorry this comment is in the wrong place)